src/r3/atomic-swap/corda/evm-bridge-workflows/src/test/kotlin/com/r3/corda/evmbridge/ProofTests.kt

```diff
@@ -0,0 +1,77 @@
+package com.r3.corda.evmbridge
+
+import com.r3.corda.cno.evmbridge.dto.TransactionReceipt
+import com.r3.corda.evmbridge.internal.TestNetSetup
+import com.r3.corda.evmbridge.workflows.*
+import net.corda.core.utilities.getOrThrow
+import org.junit.Test
+import org.web3j.crypto.Hash
+import kotlin.test.assertNotNull
+import com.r3.corda.interop.evm.common.trie.PatriciaTrie
+import org.web3j.rlp.RlpEncoder
+import org.web3j.rlp.RlpString
+import org.web3j.utils.Numeric
+import kotlin.test.assertEquals
+
+class ProofTests : TestNetSetup() {
+
+    private val eventSignature = "Transfer(address,address,uint256)"
+    private val signatureHash: String = Numeric.toHexString(Hash.sha3(eventSignature.toByteArray()))
+    private val zeroAddress = "0".repeat(64)
+
+    override fun onNetworkSetup() {
```

Comment:
is this needed?

Reply:
If you are referring to the `onNetworkSetup`, then it was inadvertently included in the code as a residue from the base test code I was working with. This isn't necessary for the current test so I will make sure to remove it. Thanks for pointing this out! Please let me know if you were referring to something else, so I can provide a more specific answer.